src/cdk-experimental/menu/menu-item-trigger.ts

```diff
@@ -24,29 +42,121 @@ import {CdkMenuItem} from './menu-item';
   exportAs: 'cdkMenuTriggerFor',
   host: {
     'aria-haspopup': 'menu',
+    '[attr.aria-expanded]': 'isSubmenuOpen() || null',
   },
 })
-export class CdkMenuItemTrigger implements AfterContentInit {
+export class CdkMenuItemTrigger implements OnDestroy {
   /** Template reference variable to the menu this trigger opens */
   @Input('cdkMenuTriggerFor') _menuPanel?: CdkMenuPanel;
 
+  /** Emits when the attached submenu is requested to open */
+  @Output() readonly opened: EventEmitter<void> = new EventEmitter();
+
+  /** Emits when the attached submenu is requested to close  */
+  @Output() readonly closed: EventEmitter<void> = new EventEmitter();
```

Comment:
I think these outputs should have a prefixes, like `cdkMenuOpened` / `cdkMenuClosed`. 

If we were _really_ technical, the names would be e.g. `cdkMenuTriggerOpened`, but I don't think this actually makes sense.